extensions/spark/kyuubi-spark-authz/src/main/scala/org/apache/kyuubi/plugin/spark/authz/ranger/RuleReplaceShowObjectCommands.scala

```diff
@@ -58,3 +60,28 @@ case class FilteredShowTablesCommand(delegated: RunnableCommand)
 
   override def withNewChildrenInternal(newChildren: IndexedSeq[LogicalPlan]): LogicalPlan = this
 }
+
+case class FilteredShowDatabasesCommand(delegated: ShowNamespaces) extends RunnableCommand
+  with WithInternalChild {
+
+  override val output: Seq[Attribute] = delegated.output
+
+  override def run(spark: SparkSession): Seq[Row] = {
+    val catalog = spark.sessionState.catalog
```

Comment:
ShowNamespaces's logical plan is ShowNamespacesExec, I don't know whether the result is same with your logical. you can add some test for catalog of datasourcev2, for example, iceberg or hudi.

Reply:
this change will ignore DSv2 table lookup

Reply:
Thanks. Currently my testing is based on Hive Metadata, other data lake components need to be tested. If you have time, help with the testing.